content/api/tracing/code_snippets/send_trace.sh

```diff
@@ -11,7 +11,7 @@ START=$(date +%s%N)
 sleep 2
 
 # Stop the timer.
-DURATION=$(($(date +%s%N) - $START))
+DURATION= $(($(date +%s%N) - $START))
```

Comment:
Need to revert this. The space after the = is not allowed and results in a `command not found` error.

Reply:
👍